robottelo/host_helpers/api_factory.py

```diff
@@ -47,6 +48,7 @@ def make_http_proxy(self, org, http_proxy_type):
                 username=settings.http_proxy.username,
                 password=settings.http_proxy.password,
                 organization=[org.id],
+                content_default_http_proxy=True,
```

Comment:
I'm not sure if we want to always do this on proxy create via `api_factory.make_http_proxy()`, this fixture never done this before since that option is new. What is the reason to do so now?

Reply:
@vsedmik Yes, the fixture behaviour is still same, if we make it non-default then we had to configure the setting explicitely and that step we have removed it from this setup fixture, and this new option configures the setting on create